subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -1214,7 +1215,7 @@ static void s_start(void *o)
 		return;
 	}
 
-	if (!hawkbit_get_device_identity(s->device_id, DEVICE_ID_HEX_MAX_SIZE)) {
+	if (!hawkbit_get_device_identity(s->device_id, DEVICE_ID_MAX_SIZE)) {
```

Comment:
```suggestion
	if (!hawkbit_get_device_identity(s->device_id, DEVICE_ID_HEX_MAX_SIZE)) {
```

DEVICE_ID_MAX_SIZE is no longer defined